### ISSUE TITLE
Pitch and speed dialog: controls should have access keys

### ISSUE THREAD

**Issue by DavidBailes** [audacity/audacity#6149]
### Steps to reproduce

1. Select a clip.
2. Open the pitch and speed dialog using the keyboard by pressing Ctrl + Shift + P.

### Expected behavior

The controls in the dialog have access keys

### Actual behavior

They don't.

### Audacity Version

current master / alpha / nightly build

### Operating system

Windows

**Comment by chinakov** [comment]
Tested, thanks!
